1-org/folders.tf

```diff
@@ -20,7 +20,7 @@
 
 resource "google_folder" "common" {
   display_name = "common"
-  parent       = "organizations/${var.org_id}"
+  parent       = var.dev_folder != "" ? "folders/${var.dev_folder}" : "organizations/${var.org_id}"
```

Comment:
One suggestion: instead of putting this logic throughout the codebase I'd suggest consolidating it in a local:

```
locals {
  parent = var.dev_folder != "" ? "folders/${var.dev_folder}" : "organizations/${var.org_id}"
```

Reply:
+1 this will make it a lot easier to refactor